packages/table/src/Cell.tsx

```diff
@@ -4,15 +4,18 @@ import { css, cx } from '@leafygreen-ui/emotion';
 import { getCommonCellStyles } from './styles';
 import { useFontSizeContext } from './FontSizeContext';
 import { useDarkModeContext } from './DarkModeContext';
+import { palette } from '@leafygreen-ui/palette';
 
 export const tdInnerDiv = createDataProp('td-inner-div');
 
 interface HeaderCellProps
   extends HTMLElementProps<'th', HTMLTableHeaderCellElement> {
   isHeader: true;
+  isZebra?: boolean;
```

Comment:
🤔  I'm not sure `isZebra` is the best prop name for this. If you're not familiar with the phrase "zebra striping" it seems a little silly 😆. I'd go with something a little more clear, like `alternateRowHighlightColor` (though maybe not that verbose)

Reply:
Alternatively, we could do what we do with the Select Options, and expose a different API than we use in the child we render in `React.Children.map`

Reply:
HAHA yeah you're right. Renaming to `isAltColor`